.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules
-/dist
\ No newline at end of file
+/dist
+*lock*
```

Comment:
As they are a vital part of production, I think it's better to leave the lock files in the project that gets pushed up to GitHub

I would otherwise like to see .DS_Store in all the .gitignore files.